packages/compass-e2e-tests/tsconfig.json

```diff
@@ -2,6 +2,6 @@
   "extends": "@mongodb-js/tsconfig-compass/tsconfig.common.json",
   "compilerOptions": {
     // https://webdriver.io/docs/typescript/#framework-setup
-    "types": ["node", "@wdio/globals/types"]
+    //"types": ["node", "@wdio/globals/types"]
```

Comment:
Intentional/related?

Reply:
webdriverio says to do that but it works fine in vscode and in reality without and with it I can't run `npx tsc -p tsconfig-lint.json --noEmit`. So I might just remove it.

Reply:
I find their typing odd. WebdriverIO is this namespace that's just available without importing anything.